src/com/chschmid/jdotxt/gui/controls/JdotxtTaskPanel.java

```diff
@@ -43,6 +45,16 @@
 
 @SuppressWarnings("serial")
 public class JdotxtTaskPanel extends JPanel {
+
+	private static final Map<Priority,Color> BACKGROUND_COLOR = new EnumMap<Priority, Color>(Priority.class){{
+		put(Priority.A, new Color(230,50,0));
+		put(Priority.B, new Color(255,140,0));
+		put(Priority.C, new Color(255,200,0));
+		put(Priority.D, new Color(230,230,0));
+		put(Priority.E, new Color(100,200,100));
+		put(Priority.NONE, new Color(220,220,220));
+	}};
+
```

Comment:
Could not find better place in the code, so put it here.

Not sure if you can reproduce it.  I observe this:  launch the app, then move from one display to another (using `Win + Shift + Arrow Key`), and priority input fields disappear, and I cannot find a way to get them back:

![image](https://user-images.githubusercontent.com/8833179/93343705-7bad4a00-f839-11ea-88e3-69fa7c5d915a.png)

Further observations:

* After I switch to another todo.txt and then back (while still on the 'bad' display) -- they re-appear.
* When in a 'bad' state, checkbox and delete buttons are also not available.
* The displays I use are slightly different size, this might be the reason -- seems like resize event is not handled properly when moving from one display to another.
* Window was maximized in this test.